saleor/static/dashboard-next/AppRoot.tsx

```diff
@@ -93,6 +93,7 @@ const decorate = withStyles(
     drawerDesktop: {
       backgroundColor: "transparent",
       borderRight: "0 none",
+      height: `calc(100vh - ${64 + theme.spacing.unit * 2}px)`,
```

Comment:
Where does 64 come from?

Reply:
Navigation bar's height. Now I think it should be defined as `const`.